tests/pytests/unit/transport/test_tcp.py

```diff
@@ -335,3 +374,58 @@ def connect(*args, **kwargs):
             client.io_loop.run_sync(client.connect)
     finally:
         client.close()
+
+
+async def test_when_async_req_channel_with_syndic_role_should_use_syndic_master_pub_file_to_verify_master_sig(
```

Comment:
These long names.....

Reply:
Yeah @s0undt3ch, we need to have a talk with @garethgreenaway about naming conventions ;)

Reply:
And @waynew.